src/tests/ftest/control/dmg_nvme_scan_test.yaml

```diff
@@ -7,15 +7,8 @@ timeout: 60
 server_config:
   name: daos_server
   port: 10001
-  transport_config:
-    allow_insecure: True
-agent_config:
-  transport_config:
-    allow_insecure: True
 dmg:
   dmg_sub_command: storage
   storage:
     storage_sub_command: scan
-  insecure: True
-  transport_config:
-    allow_insecure: True
+  insecure: False
```

Comment:
Shouldn't this have been removed as well?

Reply:
This is a sub-command parse by the test script.